src/graphics/graphics.h

```diff
@@ -85,6 +90,10 @@ class GraphicsManager : public Common::Singleton<GraphicsManager>, public Events
 	/** Enable/Disable face culling. */
 	void setCullFace(bool enabled, GLenum mode = GL_BACK);
 
+	/** Configure scaling for the GUI */
```

Comment:
Doxygen comments can't span multiple methods, unfortunately. You need to add one for each. (And end those comments with a fullstop, please.)